pkg/api/config/config.go

```diff
@@ -121,16 +121,23 @@ type SchedulerConfig struct {
 	NumWorkers int
 }
 
+type LDAPCredentials struct {
+	BindDN       string
+	BindPassword string
+}
+
 type LDAPConfig struct {
+	CredentialsFile    string
+	Unauthenticated    bool
 	Port               int
 	Insecure           bool
 	StartTLS           bool // if !Insecure, then StartTLS or LDAPs
 	SkipVerify         bool
 	SubtreeSearch      bool
 	Address            string
-	BindDN             string
+	BindDN             string `json:"-"`
```

Comment:
Remove these completely from here.

Reply:
We can't remove them. We need a way to access their value from the config. How can we transmit their values through the config if we don't set them here?

Reply:
Can it become lowercase so limited in scope (or) put it behind Getter methods.